pom.xml

```diff
@@ -46,24 +36,35 @@
         <developer>
             <name>Chris Cladden</name>
         </developer>
+        <developer>
+            <name>Bryan Turner</name>
+            <email>incommand@hotmail.com</email>
+        </developer>
     </developers>
 
+    <scm>
+        <connection>scm:git:git@github.com:brettwooldridge/NuProcess.git</connection>
+        <developerConnection>scm:git:git@github.com:brettwooldridge/NuProcess.git</developerConnection>
+        <url>git@github.com:brettwooldridge/NuProcess.git</url>
+    </scm>
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+    </distributionManagement>
+
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
-    <parent>
-        <groupId>org.sonatype.oss</groupId>
-        <artifactId>oss-parent</artifactId>
```

Comment:
Per the Central documentation, `oss-parent` is unmaintained and shouldn't be used